wnghub/controller/view.py

```diff
@@ -54,11 +72,33 @@ def _display_table(self, headers, notifications_table):
     def _write_stdout(self, str_to_write: str):
         pass
 
+    def _remove_fields_for_terminal_size(self, headers, fields):
```

Comment:
I think if we fix a problem I introduced with this originally, we can make this more general.

This is honestly how I should've defined each attribute (rather than an arbitrary pair)
```py3
@dataclass
class Attribute:
    field_name: str
    header: str
    min_size: Optional[int] = None
```

And then `_default_attributes` could look like
```py3
_default_attributes = [
    Attribute('abbrev_title', 'Title'),
    Attribute('html_url', 'Url',  min_size = 90),
    // ...
]
```

Then the filtering method would just be 'if a min size is set, and it is not met, remove the attribute from the attributes that you return.

Also: I think attributes should be filtered out prior to converting them into headers and fields.

Reply:
Did this. had to change unpack_attributes slightly, but for the most part everything could remain the same.